tmc-langs-rust/src/test/resources/testCheat/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+
+pub fn mul_xor_add(a: u64, b: u64) -> u64 {
+    a + b
+}
+
+#[test]
+fn it_shall_work() {
+
```

Comment:
Test fails when there is a panic (that wasn't expected). In this case I am testing that if student adds their own test that has same name as test for the exercise that it doesn't break.